src/itest/java/com/iexec/sms/ApiClient.java

```diff
@@ -27,14 +27,14 @@
     String API_URL = "http://localhost:${local.server.port}";
 
     @PostMapping("/apps/{appAddress}/secrets/{secretIndex}")
-    ResponseEntity<String> addApplicationRuntimeSecret(
+    ResponseEntity<String> addAppDeveloperRuntimeSecret(
```

Comment:
```suggestion
    ResponseEntity<String> addAppDeveloperAppRuntimeSecret(
```

Reply:
Some thoughts about that: are we sure we want our item to be named `AppRuntimeSecret` instead of `RuntimeSecret`? Would the last one be confusing?
I personally think that `AppDeveloperRuntimeSecret[XXX]`, `AppRequesterRuntimeSecret[XXX]`, ... are explicit enough on what we expect to find in those classes: something related to apps and runtime secrets. If you still find it unclear, maybe we could just reverse the two first words: `DeveloperAppRuntimeSecret[XXX]`, `RequesterAppRuntimeSecret[XXX]`, ... This way, we keep more or less concise wording.

However, if you're sure about the naming, we'll have to rename all of our classes to conform to this naming.